figures/tasks.py

```diff
@@ -380,4 +380,7 @@ def run_figures_monthly_metrics():
     """
     logger.info('Starting figures.tasks.run_figures_monthly_metrics...')
     for site in get_sites():
-        populate_monthly_metrics_for_site.delay(site_id=site.id)
+        populate_monthly_metrics_for_site.apply_async(
+            kwargs={'site_id': 'edx.lms.core.high'}, # TODO: put in settings
```

Comment:
Thanks @melvinsoft! Please keep in mind that Figures community compatibility needs to be maintained. I think this line breaks it. 

Also `site_id` == `edx.lms.core.high` seems incorrect.

Reply:
@OmarIthawi That error is resolved. Yes, I know we'll need to think about community as well. This is one of the default Tahoe workers, but still, it should be controled by settings, I just need some directions from John to know how to load Figures settings in the tasks.py file.
I'll wait for John approval and directions before merging of course.

Reply:
It is also our enterprise servers that we need to support too. Question @melvinsoft or @OmarIthawi , is `edx.lms.core.high` a queue that exists in upstream Open edX?

Reply:
@johnbaldwin Yes, people can change it when they deploy, but those are the default ones: https://github.com/edx/configuration/blob/open-release/juniper.master/playbooks/roles/edxapp/defaults/main.yml#L653

Question, how can I include Figures settings in this file? I'd like to use the same settings I'm using in the lms_settings.py file.

Reply:
@melvinsoft If I understand your question, right, you are asking about how to have `figures.tasks` read the value 'edx.lms.core.high' into the `apply_async` function?

If so, there isn't a "best practices" way now, but I'll give you a solution, and interested in comments from @OmarIthawi. So, first some history is important.

Way back in Figures early days, Figures had values injected into `figures.settings`. This was the old way: https://github.com/appsembler/figures/blob/0.2.0/figures/settings.py

But @OmarIthawi didn't like and I saw his point, so when he added Hawthorn plugin support, this value injection from `django.conf.settings.ENV_TOKENS['FIGURES'] into `figures.settings` went away. See here: https://github.com/appsembler/figures/pull/84

After this update, we didn't need Figures to even read from `ENV_TOKENS['FIGURES'] until Omar recently added site filtering on active sites. See these two:

1. https://github.com/appsembler/figures/pull/321
2. https://github.com/appsembler/figures/blob/master/figures/sites.py#L312

So there is where we stand today. Here's one option:

1. Set a default value for `FIGURES_MONTHLY_METRICS_QUEUE` in `figures/settings/__init__.py` or `figures/settings/lms_production.py` (I don't have a really strong opinion here. There is no option for NOT 'lms_production' module, so _everything_ in figures settings is 'lms_production')

2. in `figures.tasks` do something similar to what Omar did in `figures.sites`. Here's one option

```
from figures.settings.lms_production import FIGURES_MONTHLY_METRICS_QUEUE
```
Then in the task function that calls an `apply_async` method with the queue option:

```
queue = settings.ENV_TOKENS['FIGURES'].get('FIGURES_MONTHLY_METRICS_QUEUE',
    FIGURES_MONTHLY_METRICS_QUEUE)
populate_monthly_metrics_for_site.apply_async(site_id=site.id, queue=queue)
```